app/components/feeds/FeedSelectorModal.tsx

```
@@ -0,0 +1,214 @@
+            <div className={styles.previewingFeedCard}>
+              <PreviewingFeedCard
+                feedTitle={previewFeed.name}
+                feedId={selectedFeedId}
+              />
+            </div>
+          )}
+
+          {/* Shows "Back to your feeds" or "Back to all threads" */}
+          {backNavigation && (
+            <div
+              role="button"
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)